### PR TITLE
add state in oauth url

### PR DESCRIPTION
**Pull request by SimonLab**
ref: #33

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -2,7 +2,7 @@ defmodule ElixirAuthGoogle.MixProject do
   use Mix.Project
 
   @description "Minimalist Google OAuth Authentication for Elixir Apps"
-  @version "1.0.3"
+  @version "1.0.4"
```
Comment (nelsonic) [inline mix.exs]: Let's make this match the version in `elixir-auth-github` considering the feature parity. 💭 
```suggestion
  @version "1.1.0"
```

File: test/elixir_auth_google_test.exs [hunk 2]
```diff
@@ -30,6 +30,15 @@ defmodule ElixirAuthGoogleTest do
     assert ElixirAuthGoogle.generate_oauth_url(conn) =~ "https://accounts.google.com/o/oauth2/v2/auth?response_type=code"
   end
 
+  test "get Google login url with state" do
+    Application.put_env(:elixir_auth_google, :google_client_id, 123)
```
Comment (nelsonic) [inline test/elixir_auth_google_test.exs]: @SimonLab are we using this?

Comment (SimonLab) [reply test/elixir_auth_google_test.exs]: Good catch, we indeed don't use it anymore. I'm removing this now.

File: test/elixir_auth_google_test.exs [hunk 3]
```diff
@@ -22,14 +22,21 @@ defmodule ElixirAuthGoogleTest do
   end
 
   test "get Google login url" do
-    Application.put_env(:elixir_auth_google, :google_client_id, 123)
```
Comment (nelsonic) [inline test/elixir_auth_google_test.exs]: Also removed it from this older test as no longer required. (_that way it won't confuse us in the future ..._) 👍

File: test/elixir_auth_google_test.exs (hunk 2 above)
Comment (nelsonic) [reply test/elixir_auth_google_test.exs]: Cool. thanks for confirming. 👍

**Comment by nelsonic** [comment]
Package published to https://hex.pm/packages/elixir_auth_google/1.1.0 🚀

**Pull request merged**